algorithms/C/tree/MaxElementOfTree.c

```diff
@@ -0,0 +1,119 @@
+// Wap to find the maximum(using recursion) element in a binary search tree.

```

Comment:
Please change `Wap` to `Way` and write `using recursion` as written below
> // Way to find the maximum element in a binary search tree (using recursion)

Reply:
@UG-SEP here's a quick tip. 
if you want to suggest direct changes, you directly add changes suggestion in your comment.

there is an option to do so here 👇
<br/>
<img width="799" alt="Screenshot 2021-04-21 at 8 53 37 AM" src="https://user-images.githubusercontent.com/54987647/115492451-76a4f300-a27f-11eb-981b-6d8efcdda34d.png">

Reply:
Thanks for  telling me @Arsenic-ATG